### PR TITLE
karaf-camel-amq: add parameters to configure AMQ service name, user n…

### PR DESCRIPTION
**Pull request by lburgazzoli**
…ame and password, remove fabric8 amq component and use fabric8-karaf for property resolution

### CODE REVIEW

**Comment by lburgazzoli** [comment]
@davsclaus can you review / merge ?

File: quickstart/karaf/camel-amq/src/main/resources/OSGI-INF/blueprint/camel-context.xml [hunk 1]
```diff
@@ -23,14 +23,25 @@
...
            xsi:schemaLocation="
              http://www.osgi.org/xmlns/blueprint/v1.0.0 https://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
              http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
 
+  <ext:property-placeholder evaluator="camel"/>
 
   <bean id="orderGenerator" class="io.fabric8.quickstarts.camelamq.OrderGenerator"/>
 
+  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="brokerURL" value="tcp://${env+service:ACTIVEMQ_SERVICE_NAME}"/>
```
Comment (davsclaus) [inline quickstart/karaf/camel-amq/src/main/resources/OSGI-INF/blueprint/camel-context.xml]: What karaf version requires this new env+service stuff ?

Comment (lburgazzoli) [inline quickstart/karaf/camel-amq/src/main/resources/OSGI-INF/blueprint/camel-context.xml]: It is handled by an aries blueprint extender provided by fabric8-karaf

**Comment by davsclaus** [comment]
There is a merge conflict, you would need to rebase

**Comment by lburgazzoli** [comment]
@davsclaus Rebased

**Comment by davsclaus** [comment]
ok to test

**Comment by davsclaus** [comment]
[merge]

**Pull request merged**